Mage.Tests/src/test/java/org/mage/test/cards/abilities/keywords/GiftTest.java

```diff
@@ -0,0 +1,97 @@
+package org.mage.test.cards.abilities.keywords;
+
+import mage.abilities.keyword.HexproofAbility;
+import mage.abilities.keyword.IndestructibleAbility;
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * @author TheElk801
+ */
+public class GiftTest extends CardTestPlayerBase {
```

Comment:
can one of the tests cover the token creation gift option as well?

Reply:
yeah I was waiting for a card with a more easy to test effect

Reply:
![image](https://github.com/user-attachments/assets/1c489f47-2c9d-46d0-858f-ced60c89bd22)
speak of the devil